CONTRIBUTING.md

```diff
@@ -13,15 +13,18 @@ If you are stuck, please feel free to ask any questions or ask for help.
 [Quick links](#quick-links)
 
 [How can I contribute?](#how-can-i-contribute)
-  * [Bug reporting](#bug-reporting)
-  * [Suggesting enhancements](#suggesting-enhancements)
-  * [Your first code contribution](#your-first-code-contribution)
-  * [Pull requests](#pull-requests)
+
+- [Bug reporting](#bug-reporting)
+- [Suggesting enhancements](#suggesting-enhancements)
+- [Dev Environments](#dev-environments)
+- [Your first code contribution](#your-first-code-contribution)
+- [Pull requests](#pull-requests)
```

Comment:
```suggestion
- [Bug reporting](#bug-reporting)
- [Suggesting enhancements](#suggesting-enhancements)
- [Your first code contribution](#your-first-code-contribution)
- [Pull requests](#pull-requests)
- [Dev Environments](#dev-environments)
```

I think this contribution is important, but I'd like to avoid overwhelming a newcomer to this project. It's possible to contribute to this codebase without dev environments (e.g. documentation PR), so listing this last would be my preference.

Reply:
@kenibrewer - can you also make the necessary changes to restructure the order below?